README.md

```diff
@@ -1,36 +1,184 @@
 =================
 
-**⚠️ This is a work in progress ⚠️**
-
-Tools to "brand" Origami components. `o-brand` is intended as an internal tool and is not recommended for use outside of Origami components.
+Tools to "brand" Origami components. Other non-Origami projects should not depend on `o-brand` directly.
 
 - [Terms](#terms)
 - [Usage](#usage)
-	- [Sass](#sass)
 - [Contact](#contact)
 - [Licence](#licence)
 
 ## Terms
 
 ### Brand
 
-A brand is a collection of configuration to tailor a component for specifc usecases. A branded Origami component is able to output the correct visual style and features for a requested brand.
+A brand is a collection of configuration to tailor a component for specific use-cases. A branded Origami component is able to output differing styles for a requested brand. It is also able to include extra styles specific to a brand.
 
 Examples of brands include:
 
 - Master: FT branding for public ft.com sites and affiliates.
-- Internal: A minimal style with subtle branding for internal products, tools, and documentation.
+- Internal: Style suitable for internal products, tools, and documentation.
 - Whitelabel: Base, structural styles only.
 
-### Variants
-
-A branded component may contain a number of variants e.g. a button component may provide a "big" and "small" variant. Variants may modify the appearance or functionality of a component. They may override default brand styles, e.g. change the colour of a button. Variants may also require new markup e.g. to add a breadcrumb to a header component. Variants must be optional and build upon a fully functional branded component.
+### Variant
+
+A branded component may contain one or more variants e.g. a button component may provide a "big" and "small" variant. Variants may change the appearance or functionality of a component. They may override default brand styles, e.g. change the colour of a button. Variants may also require new markup e.g. to add a breadcrumb to a header component. Variants must be optional and build upon a fully functional branded component.
+
+E.g.
+
+1. Brand variants modify a fully functional foundation.
+```scss
+	.o-example {
+		content: 'component';
+	}
+```
+```html
+	<div class="o-example"></div>
+```
+2. Variants may override styles.
+```scss
+	.o-example--inverse {
+		content: 'example variant "inverse"';
+	}
+```
+```html
+	<div class="o-example o-example--inverse"></div>
+```
+3. Variants may require additional markup.
+```scss
+	.o-example__extra {
+		content: 'example feature variant "extra"';
+	}
+```
+```html
+	<div class="o-example o-example--inverse">
+		<div class="o-example__extra"></div>
+	</div>
+```
 
 ## Usage
 
 Mixins within `o-brand` help configure components to support brands. There is no configuration in `o-brand`. It provides the mechanisms for components to apply their own brand support.
 
+The following mixins and functions help brand a component.
+
+- oBrandDefine
```

Comment:
What do we think about having these link to their documentation?

Reply:
Good shout. I'll add them after the first release rather than guess a url.